<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcPtosPrimordialesDialog</class>
 <widget class="QDialog" name="calcPtosPrimordialesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="desdeLabel">
       <property name="text">
        <string>Desde: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="desdeDateEdit">
       <property name="date">
        <date>
         <year>1900</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="hastaLabel">
       <property name="text">
        <string>Hasta: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="hastaDateEdit">
       <property name="date">
        <date>
         <year>2020</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="planetaLabel">
       <property name="text">
        <string>Planeta:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="planetaComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="signoLabel">
       <property name="text">
        <string>Signo: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="signoComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="gradosDeSignoLabel">
       <property name="text">
        <string>Grados de Signo:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="gradosDeSignoDoubleSpinBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="toleranciaLabel">
       <property name="text">
        <string>Tolerancia: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="toleranciaDoubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>calcPtosPrimordialesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>calcPtosPrimordialesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
